docs/Marketplace.md

```diff
@@ -0,0 +1,360 @@
+Password:
+Repeat password:
+
+Your new key was generated
+
+Public address of the key:   0x93976895c4939d99837C8e0E1779787718EF8368
+...
+```
+
+In this example, the public address of the signer account is `0x93976895c4939d99837C8e0E1779787718EF8368`. Yours will print a different address; write it down.
+
+### 1.2. Configure The Network and Create the Genesis Block
+
+The next step is telling geth what kind of network you want to run. We will be running a [pre-merge](https://ethereum.org/en/roadmap/merge/) network with Proof-of-Authority consensus. To get that working, create a `network.json` file with the following content:
+
+```json
+{
+  "config": {
+    "chainId": 12345,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "arrowGlacierBlock": 0,
+    "grayGlacierBlock": 0,
+    "clique": {
+      "period": 1,
+      "epoch": 30000
+    }
+  },
+  "difficulty": "1",
+  "gasLimit": "8000000",
+  "extradata": "0x000000000000000000000000000000000000000000000000000000000000000093976895c4939d99837C8e0E1779787718EF83680000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "alloc": {
+    "0x93976895c4939d99837C8e0E1779787718EF8368": {
+      "balance": "10000000000000000000000"
+    }
+  }
+}
+```
+
+Note that the signer account address is embedded in two different places:
+* inside of the `"extradata"` string, surrounded by zeroes and stripped of its `0x` prefix;
+* as an entry key in the `alloc` session.
+
+Make sure to replace that ID with the account ID that you wrote down in Step 1.1. Once that is done, you can initialize the network with:
+
+```bash
+geth init --datadir geth-data network.json
+```
+
+### 1.3. Start your PoA Node
+
+We are now ready to start our $1$-node, private blockchain. To launch the signer node, open a separate terminal on the same working directory and run:
+
+```bash
+geth\
+  --datadir geth-data\
+  --networkid 12345\
+  --unlock <signer-account-address>\
+  --nat extip:127.0.0.1\
+  --netrestrict 127.0.0.0/24\
+  --mine\
+  --miner.etherbase <signer-account-address>\
+  --http\
+  --allow-insecure-unlock
+```
+
+Note that, once again, the signer account created in Step 1.1 appears both in `--unlock` and `--allow-insecure-unlock`: do not forget to insert it there. Geth will prompt you to insert the account's password as it starts up. Once you do that, it should be able to start up and begin "mining" blocks.
+
+## 2. Set Up The Marketplace
+
+Setting up the marketplace entails: _i)_ deploying the Codex Marketplace contracts to our private blockchain, and _ii)_ provisioning accounts we will later use to buying and selling storage in the marketplace with the required token balances.
```

Comment:
Numbered bullets here are probably a bit easier to follow (for me at least). Also `buying/selling` don't match the rest of the sentence:

```suggestion
Setting up the Codex marketplace entails:

1. Deploying the Codex Marketplace contracts to our private blockchain
2. Setup Ethereum accounts we will use to buy and sell storage in the Codex marketplace 
3. Provisioning those accounts with the required token balances

```